docs/integrations/sumo-apps/security-foundations.md

```diff
@@ -0,0 +1,134 @@
+---
+id: security-foundations
+title: Sumo Logic Security Foundations App
+sidebar_label: Security Foundations
+description: Shows the most important alerts to investigate and the entities that pose the greatest risk.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/sumo-apps/SecurityAnalytics.png')} alt="Thumbnail icon" width="75"/>
+
+The Sumo Logic Security Foundations app provides tools for real-time threat detection. The app’s dashboards focus on adversarial behavior within your environment, and show you the most important alerts to investigate and the entities that pose the greatest risk. You can even drill down on the alerts and entities to view the [MITRE ATT&CK](https://attack.mitre.org/) tactics and techniques associated with them.
+
+The app supports the following security technologies:
+* Audit
+   * AWS CloudTrail
+   * Azure Audit
+   * Duo
+   * GCP Audit (limited)
+   * Linux Audit Logs
+   * Microsoft Office 365
+   * Microsoft SharePoint
+   * Okta
+   * OneLogin
+   * Windows security events
+* Endpoint security
+   * CrowdStrike
+   * Proofpoint TAP
+* Network
+   * All firewall vendors
+   * All IDS/IPS vendors
+   * Cisco Umbrella
+
+
+## Install the Sumo Logic Foundations​ app
+
+The Security Foundations app is available to Sumo Logic Enterprise Security and Enterprise Suite accounts only, and requires a Professional Services contract to install and configure. For more information about this app and to request installation, send an email to `sales@sumologic.com`, contact your account manager, or see [Requesting Sumo Logic Enterprise Apps](/docs/integrations/sumo-apps/#Requesting-Sumo-Logic-Enterprise-Apps).
```

Comment:
@jpipkin1 @oaklandersumo , typically we hyperlink email addresses; was it left unlinked intentionally?

Reply:
My bad, @kimsauce. It's fixed now.